modules/Bio/EnsEMBL/Variation/DBSQL/PhenotypeFeatureAdaptor.pm

```diff
@@ -1074,6 +1074,88 @@ sub fetch_all {
   
 }
 
+sub get_PhenotypeFeatures_by_location {
```

Comment:
Minor request that I overlooked in the first review: could we have some perldoc for this method ?
This is would also allow to have the method documented in Doxygen.